tsconfig.json

```diff
@@ -22,7 +22,7 @@
         ],
         "sourceMap": true,
         "rootDir": "src",
-        "strict": true, /* enable all strict type-checking options */
+        "strict": false, /* enable all strict type-checking options */
```

Comment:
that seems scary

Reply:
I understand and share your concern. The reason behind this is that many components that handle profiles comes from Zowe Explorer which don't have this parameter set. This yields a lot of errors and it's an issue I am planning to work in a future iteration.